soap/src/java/com/zimbra/soap/mail/type/FreeBusySlot.java

```diff
@@ -57,10 +120,59 @@ public FreeBusySlot(long startTime, long endTime) {
     public long getStartTime() { return startTime; }
     public long getEndTime() { return endTime; }
 
-    public MoreObjects.ToStringHelper addToStringInfo(MoreObjects.ToStringHelper helper) {
+    public String getId() {
+		return id;
+	}
+
+	public String getSubject() {
+		return subject;
+	}
+
+	public String getLocation() {
+		return location;
+	}
+
+	public boolean isMeeting() {
+		return isMeeting;
+	}
+
+	public boolean isRecurring() {
+		return isRecurring;
+	}
+
+	public boolean isException() {
+		return isException;
+	}
+
+	public boolean isReminderSet() {
+		return isReminderSet;
+	}
+
+	public boolean isPrivate() {
+		return isPrivate;
+	}
+
+	public boolean isHasPermission() {
+		return hasPermission;
+	}
+
+	public void setHasPermission(boolean hasPermission) {
+		this.hasPermission = hasPermission;
+	}
+
```

Comment:
You should generate getters/setters for all of the new properties, it will make things easier when using this model in other places.